scripts/aws/iam/create_aws-elasticbeanstalk-service-role.sh

```diff
@@ -0,0 +1 @@
+aws iam create-role --role-name aws-elasticbeanstalk-service-role --assume-role-policy-document file://elasticbeanstalk.amazonaws.com_trusted.json
```

Comment:
With file://file.json you have to run the script from the folder `script/aws/iam`. It could be better to script something around configuring the right paths.